feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/staking/pool/SelectingNominationPoolInteractor.kt

```diff
@@ -0,0 +1,19 @@
+package io.novafoundation.nova.feature_staking_impl.domain.staking.pool
+
+import io.novafoundation.nova.feature_staking_impl.data.StakingOption
+import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.model.NominationPool
+import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.model.apy
+import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.pools.NominationPoolProvider
+import kotlinx.coroutines.CoroutineScope
+
+class SelectingNominationPoolInteractor(
+    private val nominationPoolProvider: NominationPoolProvider
+) {
+
+    suspend fun getSortedNominationPools(stakingOption: StakingOption, coroutineScope: CoroutineScope): List<NominationPool> {
+        val nominationPools = nominationPoolProvider.getNominationPools(stakingOption, coroutineScope)
+        val comparator = compareByDescending<NominationPool> { it.apy }
+            .thenByDescending { it.membersCount }
```

Comment:
We should display Nova pools first. This is already done in `RealNominationPoolRecommendator` by the way. Ideally we should use it for recomenndations to avoid dublicating logic. There is a differente between automatic recomendation and pool list though - we still want to display full pools. However it would be nice to have all recomendation logic in one place I think